ykcapi/src/lib.rs

```diff
@@ -218,6 +268,7 @@ pub extern "C" fn yk_stopgap(
             SMLocation::Indirect(reg, off, size) => {
                 let addr = unsafe { registers.get(*reg) as *mut u8 };
                 let addr = unsafe { addr.offset(isize::try_from(*off).unwrap()) };
+                //println!("direct addr: {:?}", addr);
```

Comment:
debugging left in here.

Reply:
Fixed in [c45e5e6](https://github.com/ykjit/yk/pull/578/commits/c45e5e644b2bc6b43c9d456a8192a5cc3ddbaff8)